lib/archethic_web/api/ecto_schemas/function_call_payload.ex

```diff
@@ -0,0 +1,18 @@
+defmodule ArchethicWeb.API.FunctionCallPayload do
+  @moduledoc false
+  alias ArchethicWeb.API.Types.Address
+
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  embedded_schema do
+    field(:contract, Address)
+    field(:function, :string)
```

Comment:
You should endure args is an array otherwise it will crash later.
Tried `args: "something"` and it crashed